include/psa/crypto_extra.h

```diff
@@ -572,8 +572,8 @@ psa_status_t psa_get_key_domain_parameters(
 /** \defgroup psa_tls_helpers TLS helper functions
  * @{
  */
-
-#if defined(MBEDTLS_ECP_LIGHT)
+#include <mbedtls/pk.h>
+#if defined(MBEDTLS_PK_HAVE_ECC_KEYS)
 #include <mbedtls/ecp.h>
 
 /** Convert an ECC curve identifier from the Mbed TLS encoding to PSA.
```

Comment:
I agree. I will move it close to `mbedtls_ecc_group_of_psa` in `psa_crypto.c`.
However I have a question here: since `psa_crypto.c` is already a pretty huge file and these functions are not really for PSA crypto core, but rather for providing some help for EC, what about moving both of them to `psa_utils.c`?

Reply:
Good idea. If you don't mind, in order to avoid conflicts, I'll do it in the pending PR I have that reworks `psa_util.h`: #7716

Reply:
Sure, it's OK for me ;)